### PR TITLE
refine readme

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
1. add the link to chaos-mesh
2. add features
3. add the steps to use chaos-mesh-action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,66 @@
...
+      uses: chaos-mesh/chaos-mesh-actions@master
+      env:
+        CFG_BASE64: ${CFG_BASE64}
 ```
 
-`chaos.yaml` is your chaos config file.
+For the complete configuration file, see the [sample configuration file](https://github.com/chaos-mesh/chaos-mesh-actions/blob/master/.github/workflows/chaos.yml).
 
-## Limits
+## Limitation
```
Comment (dcalvin) [inline README.md:69]:
```suggestion
## Limitations
```

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,71 +1,72 @@
-# chaos-mesh-actions ![Chaos](https://github.com/chaos-mesh/chaos-mesh-actions/workflows/Chaos/badge.svg)
+# chaos-mesh-action ![Chaos](https://github.com/chaos-mesh/chaos-mesh-action/workflows/Chaos/badge.svg)
 
-Using Chaos Mesh in Github Action.
+A GitHub Action that applies chaos enginnering by Chaos Mesh in the workflow. More detail about Chaos Mesh can see [https://chaos-mesh.org/](https://chaos-mesh.org/).
```
Comment (dcalvin) [inline README.md:3]: A GitHub action that applies chaos engineering in the development workflow using Chaos Mesh. For more information on Chaos Mesh, see [https://chaos-mesh.org/](https://chaos-mesh.org/).

File: README.md [hunk 3]
```diff
@@ -1,71 +1,72 @@
...
 
-## How To Use
+## Features
 
-`chaos-mesh-actions` can be used in GitHub's workflows, there is an example config file:
+`chaos-mesh-action` automatically deploy the Chaos Mesh environment and injects the specified chaos experiment.
```
Comment (dcalvin) [inline README.md:7]: This seems like the description for Chaos Mesh, not chaos-mesh-action

File: README.md [hunk 4]
```diff
@@ -1,71 +1,72 @@
...
-      run: |
-        kubectl get pods -n chaos-testing
-        sleep 5
-        kubectl get pods -n busybox
-        
-    - name: Run chaos mesh action
-      uses: chaos-mesh/chaos-mesh-actions@master
-      env:
-        CFG_BASE64: ${CFG_BASE64}
+## Example Usage
```
Comment (dcalvin) [inline README.md:9]: Sample usage

**Pull request merged**